setup.sh fails

Was running setup.sh it fails with:
``
>>> 3/5 Building web server <<<
npm ERR! code E403
npm ERR! 403 403 Forbidden - GET https://registry.npmjs.intuit.com:443/artifactory/api/npm/npm-intuit/xdg-basedir/-/xdg-basedir-3.0.0.tgz
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy, or
npm ERR! 403 on a server you do not have access to.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxxxx\AppData\Local\npm-cache\_logs\2021-03-25T11_44_35_458Z-debug.log
``

--- a/setup.sh
+++ b/setup.sh
@@ -20,6 +20,7 @@ cp -v target/maven-build-scanner-1.0.0-SNAPSHOT-jar-with-dependencies.jar $(mvn 
 cd server
 
 status "3/5 Building web server"
+npm update
 npm install
 
 

